ui/src/CMakeLists.txt

```diff
@@ -20,7 +20,6 @@ else()
 endif()
 
 if((((QT_VERSION_MAJOR LESS 5)) AND (MACOS)))
```

Comment:
Qt4 is no longer supported
In case, leave portaudio stuff commented somewhere but I doubt I will pick it up again